lib/plugins/digging.js

```diff
@@ -242,13 +242,16 @@ function inject (bot) {
     return block.digTime(
       type,
       creative,
-      bot.entity.isInWater,
+      _getBlockAtEyeLevel()?.name === 'water',
       !bot.entity.onGround,
       enchantments,
       bot.entity.effects
     )
   }
 
+  _getBlockAtEyeLevel = () => bot.entity.position && bot.blockAt(bot.entity.position.offset(0, bot.entity.eyeHeight, 0))
+
+  bot._getBlockAtEyeLevel = _getBlockAtEyeLevel;
```

Comment:
This can be inlined to

```js
bot._getBlockAtEyeLevel = () => bot.entity.position && bot.blockAt(bot.entity.position.offset(0, bot.entity.eyeHeight, 0))
```

if it's not being called many times

Reply:
What `  _getBlockAtEyeLevel = () => bot.entity.position && bot.blockAt(bot.entity.position.offset(0, bot.entity.eyeHeight, 0))` does is create a global variable as you are not binding it with `let` or `const`. That is causing lint to fail